builds/csharp/nuget/build_nugets.py

```diff
@@ -6,9 +6,9 @@
 import fileinput
 from typing import List
 
-version = "0.7.1.0"
-informal_version = "0.7.1.0"
-nuget_version = "0.7.1.0"
+version = "0.7.2.0"
+informal_version = "0.7.2.0-dev1"
+nuget_version = "0.7.2.0-dev1"
```

Comment:
Should it contain `-dev` versions when merging to main ?

Reply:
It should be fine, I update the version without the -dev suffix in release branches that then get merged to main and tagged with the specified version.